apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/DataSetManifest.java

```diff
@@ -475,4 +507,262 @@ public String toString() {
       return builder.toString();
     }
   }
+
+  @XmlAccessorType(XmlAccessType.FIELD)
+  public static final class SyntheaEndStateProperties {
+    @XmlElement(name = "bene_id_start", required = true)
+    protected long beneIdStart;
+
+    @XmlElement(name = "bene_id_end", required = true)
```

Comment:
changed....however, this is sort of a by-product of bad practice. In theory, we should be generating Java POJO(s) via JAXB consumption of the XSD (maven plugin). In that scenario, you don't get element-level comments.

Added doc to elements.

Reply:
Yeah if they were generated we'd skip them for documentation, but for checkstyle purposes as long as we hand-write something it'll need docs. It would be nice to switch to generating more of this boilerplate code in the future.